lib/frontman/commands/serve.rb

```diff
@@ -75,6 +75,35 @@ def serve
       FrontmanServer.set :public_folder, Frontman::Config.get(
         :public_dir, fallback: 'public'
       )
+
+      port = Frontman::Config.get(:port, fallback: 4568)
+      num_retries = Frontman::Config.get(:port_retries, fallback: 3)
+
+      port_retry_strategy = Frontman::Config.get(
+        :port_retry_strategy,
+        fallback: ->(p) {
+          port_in_use = false
+
+          (1 + num_retries).times do
+            begin
+              port_in_use = Socket.tcp('localhost', p, connect_timeout: 3) { true }
```

Comment:
Getting line length errors here again. 83/100. Would love that 100 char line limit increase 😉

Reply:
Okay, feel free to set the limit to 100, it seems to be a recurring issue indeed and if we can prevent that this way let's do that 💯